### DISCUSSION TITLE
ERROR: en-core-web-trf 3.0.0 has requirement spacy<3.1.0,>=3.0.0, but you'll have spacy 3.3.1 which is incompatible.

### DISCUSSION THREAD

**Discussion by andrew3875** [explosion/spaCy#10990] [Help: Best practices]
Ladies and gentlemen,
I have an above mentioned trouble.
Is there a way to use latest pipeline  along with the latest (say 3.3.1) Spacy release?
If yes how ?

**Comment by rmitsch** [comment]
Your version of spaCy (3.3.1) and the version of the model you are using (3.0.0) are not matching. If you download a model, such as `en-core-web-trf`, spaCy should pick the correct version automatically. Have you tried `spacy download en-core-web-trf` yet?

**Reply by andrew3875** [reply]
I haven't tried spacy download en-core-web-trf yet. The problem is that I already  use the latest model v 3.0.0.
So the problem persists. How could I work around that?

**Reply by rmitsch** [reply]
We strongly recommend using the latest model version available. There is no downside to updating your model (if you don't rely on specific predictions). Is there a particular reason why you'd want to stick with v3.0.0?

**Reply by andrew3875** [reply]
Thanks for your input. The problem is that the version of en-core-web-trf 3.0.0 is 3.0.0 and it is the _latest_ version. Or is there something that  I am missing?

**Reply by rmitsch** [reply]
v3.3.0 is available, see [the docs](https://spacy.io/models/en#en_core_web_trf) or [the GitHub release page](https://github.com/explosion/spacy-models/releases/tag/en_core_web_trf-3.3.0).
